replica/database.hh

```diff
@@ -1496,6 +1496,7 @@ public:
     std::vector<sstring> get_user_keyspaces() const;
     std::vector<sstring> get_all_keyspaces() const;
     std::vector<sstring> get_non_local_strategy_keyspaces() const;
+    std::unordered_map<sstring, locator::effective_replication_map_ptr> get_non_local_strategy_keyspaces_erms() const;
```

Comment:
_erms() is not friendly to newcomers, especially since the return type likely won't be seen at the call site. Please expand.